Modules/GLO/src/ITSTPCMatchingTask.cxx

```diff
@@ -116,6 +139,7 @@ void ITSTPCMatchingTask::startOfCycle()
 
 void ITSTPCMatchingTask::monitorData(o2::framework::ProcessingContext& ctx)
 {
+  ILOG(Info) << "********** Starting monitoring" << ENDM;
```

Comment:
We recommend never logging from `monitorData` because this method is usually called at very high rate.

Reply:
You are right, it was a leftover of my tests. Changed to debug.